docs/how-to-guides/software-management/software-support.md

```diff
@@ -1,34 +1,52 @@
 # Software Support
-This document provides information about the software support process, including scheduling, project management tools and practices, issue soliciation, issue selection, and prioritization.
 
+!!! Note "How to Submit Issues" 
+    Our code is managed [via GitHub](https://github.com/DOI-USGS)
 
-## Step 1: Scheduling the Support Sprint
+    Any issues with our software portfolio should be tracked in our repos through [GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues). Links for some our bigger repos: 
+
+    * [ISIS](https://github.com/DOI-USGS/ISIS3/issues)
+    * [ALE](https://github.com/DOI-USGS/ale/issues)
+    * [USGSCSM](https://github.com/DOI-USGS/usgscsm/issues)
+
+    This document provides information about the software support process. That is, how our development team goes about addressing these issues. Including scheduling, project management tools and practices, issue soliciation, issue selection, and prioritization.
+
+## Support Sprint Process 
+
+Support sprints are dedicated efforts, time-boxed to 3 weeks quarterly. We schedule these as part of our internal scheduling procedures, and support sprints are scheduled 3 to 6 weeks ahead of time. This includes a prioritization meeting where we go through the issues accross our repositories and put them in prioritization order. These meetings are open to the public. 
+
+??? Question "How do I get Notified of Support Sprints and prioritization meetings?" 
+    We email anyone who has opted in to our software support newsletter which includes when the support sprint was scheduled, when the prioritization meetings are happening, and a link to the meeting. These are sent shortly after it's scheduled and again right before the meeting. 
+    
+    Sign up at [here](https://public.govdelivery.com/accounts/USDOIGS/signup/39118). 
+
+### Step 1: Scheduling the Support Sprint
 While there is some leeway in the exact timing of each support sprint, the sprints must be scheduled quarterly and should be scheduled roughly 3 months apart. Project scheduling should be coordinated by the project/technical lead and the software lead during routine software scheduling meetings.
 
 After setting up the sprint schedule, the technical lead should create a software support board and communicate the schedule to contributors.
```

Comment:
Do we also want to add here that the prioritization meeting will always be scheduled a week before the support sprint starts?

Reply:
good catch, I thought I wrote that down but I probably thought it and didn't actually write it lmao